Examples/Tests/reduced_diags/inputs

```diff
@@ -0,0 +1,71 @@
+# Maximum number of time steps
+max_step = 200
+
+# number of grid points
+amr.n_cell =   32  32  32
+
+# Maximum allowable size of each subdomain in the problem domain;
+# this is used to decompose the domain for parallel calculations.
+amr.max_grid_size = 32
```

Comment:
For the sake of verification, could you run this test with
- 1 MPI rank, 1 box
- Multiple MPI ranks, multiple boxes per MPI rank

and make sure they give the same result to machine precision? 
This section should help you do that https://warpx.readthedocs.io/en/latest/running_cpp/parallelization.html/